File: app/src/main/java/com/callmangement/EHR/models/ModelLogout.java
package com.callmangement.EHR.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class ModelLogout implements Serializable {
    @SerializedName("status")
    private String status;
    @SerializedName("message")
    private String message;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/com/callmangement/model/logout/ModelLogout.kt
package com.callmangement.model.logout

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class ModelLogout : Serializable {
    @SerializedName("status")
    var status: String? = null

    @SerializedName("message")
    var message: String? = null
}
